### PR TITLE
update the issue template and README.md about support

### PR DESCRIPTION
**Pull request by neolit123**
since we are getting support tickets occasionally here, let's make it clearer in the readme and issue template that we keep this repository for feature / bug tracking only. we can still keep some `kind/support` tickets open, if we suspect there is a bug.

update the issue template to:
- include up-to-date links to the TS guide
- link to the support channels
- remove the "Is this a request for help?" question

update the README.md:
- stating that this issue tracker is not for support
- link to the support channels
- include up-to-date links to the TS guide
- add some more documentation links

### CODE REVIEW

**Comment by neolit123** [comment]
@SataQiu FYI

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
-<!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
+<!--
+!!! IMPORTANT  !!!
 
-## Is this a request for help?
+Before hitting the submit button, please note that requests for support must be send
```
Comment (pacoxu) [inline .github/ISSUE_TEMPLATE.md:4]: send -> sent

File: README.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 The purpose of this repo is to aggregate issues filed against the [kubeadm component](https://git.k8s.io/kubernetes/cmd/kubeadm).
 
+**NOTE:** This issue tracker is not designated for providing support for kubeadm users.
+Please see the [Support](#support) section bellow.
```
Comment (pacoxu) [inline README.md:6]: bellow -> below

**Comment by fabriziopandini** [comment]
lgtm pending above comments

**Comment by neolit123** [comment]
good catches @pacoxu . updated.

**Pull request merged**